doc/tutorial/gradient.rst

```diff
@@ -897,10 +897,10 @@ Here is how we can construct a variant and invoke the algorithm:
 
 .. code-block:: cpp
 
-  #include <boost/mpl/vector.hpp>
+  #include <boost/mp11.hpp>
   #include <boost/gil/extension/io/jpeg_dynamic_io.hpp>
```

Comment:
By the way, the `jpeg_dynamic_io.hpp` is not correct either and could be with `boost/gil/extension/io/jpeg/old.hpp`